iter-1/src/main/java/com/github/lette1394/App.java

```diff
@@ -13,5 +17,10 @@ public static void main(String[] args) {
     if (add.equals("1+2")) {
       System.out.println(3);
     }
+
+    if(add.equals("N+M")){
```

Comment:
commit 하시기 전에 formatter를 돌려주세요~
```suggestion
    if (add.equals("N+M")) {
```

Reply:
@lette1394 
commit 전 formatter 돌리고
요구사항 3을 만족하면서 또한 요구사항 1,2 또한 만족하도록 수정했습니다

Reply:
@Gyoung-min 
resolve 합니다.